src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/AutoInsert/RazorOnAutoInsertProviderTestBase.cs

```diff
@@ -40,7 +40,7 @@ protected void RunAutoInsertTest(string input, string expected, int tabSize = 4,
         var provider = CreateProvider();
 
         // Act
-        var edit = provider.TryResolveInsertion(position, codeDocument, enableAutoClosingTags: enableAutoClosingTags);
+        provider.TryResolveInsertion(position, codeDocument, enableAutoClosingTags: enableAutoClosingTags, out var edit);
```

Comment:
Probably should `Assert.True(..)` around this

Reply:
Thinking about it, we can actually check the returned bool value is consistent with returned edit value, like so

        Assert.True((result && edit is not null) || (!result && edit is null))

Reply:
That's not worth an assert. The C# compiler already does that for you at compile time thanks to the `[NotNullWhen(true)]`.

Reply:
That's fair, I'll go ahead and remove it then.